.github/workflows/run_tests/action.yml

```diff
@@ -4,7 +4,6 @@ runs:
     - name: Run unit tests
       run: |
           cd ${GITHUB_WORKSPACE}/build
-          ./tests/test_SVD
```

Comment:
Does this remove the running of all SVD tests?

Reply:
No it is not. This only tests the base `SVD` class, and only the time-interval related functionalities. The entire test routines become unnecessary as we remove the time interval.